### PR TITLE
Tweak battery_level

### PR DESCRIPTION
**Pull request by ben-e-whitney**
Improve documentation for obsolete `mode` and `show_percent_with_blocks` parameters. Add new parameters for controlling how the battery level is decided to be bad, degraded, or good.

### CODE REVIEW

File: py3status/modules/battery_level.py [hunk 1]
```diff
@@ -106,9 +122,12 @@ class Py3status:
     hide_seconds = False
     notification = False
     notify_low_level = False
+    threshold_bad = 10
+    threshold_degraded = 30
+    threshold_full = 100
     # obsolete configuration parameters
     mode = None
-    show_percent_with_blocks = None
+    show_percent_with_blocks = False
```
Comment (tobes) [inline py3status/modules/battery_level.py]: This is obsolete so I'd not bother to change it.

File: py3status/modules/battery_level.py [hunk 2]
```diff
@@ -53,15 +64,20 @@
...
+        - if 'format' is "{icon}" and 'mode' is "text", the `format` is set to
+          "Battery: {percent}"
+        - all other values are ignored
+        default is None
+    show_percent_with_blocks: an old way to define `format` parameter. The
+      current behavior is:
+        - if 'format' is not "{icon}", this parameter is completely ignored
+        - if 'format' is "{icon}" and 'mode' is "ascii_bar" or "text", this
+          parameter is completely ignored
+        - if the value is True, the `format` is set to "{icon} {percent}%"
```
Comment (tobes) [inline py3status/modules/battery_level.py]: Could you remove the `-` that you have added as it breaks the `modules/README.md`

**Comment by ben-e-whitney** [comment]
I have reverted the change to the default `show_percent_with_blocks` value and removed the unordered list.

**Comment by ben-e-whitney** [comment]
I meant to rebase, not add the pomodoro changes into my PR. What should I do? Sorry; new to this!

**Comment by tobes** [comment]
`git revert 07ee5357a6` will remove that commit.

It takes a bit of time to get the hang of how it all works :)

**Comment by ben-e-whitney** [comment]
OK, I have reverted that commit. Thanks @tobes.

**Comment by ultrabug** [comment]
Thanks a lot everyone, it's a nice addition :)

**Pull request merged**